store/sqlstore/command_webhook_store.go

```diff
@@ -47,7 +47,10 @@ func (s SqlCommandWebhookStore) Save(webhook *model.CommandWebhook) (*model.Comm
 		return nil, err
 	}
 
-	if err := s.GetMaster().Insert(webhook); err != nil {
+	if _, err := s.GetMasterX().NamedExec(`INSERT INTO WebHook
```

Comment:
`INSERT INTO CommandWebhooks`

Also `CreatedAt` should be `CreateAt`

Reply:
sorry for that, i'm habit to put the verb in past